### ISSUE TITLE
Feature request: Write something other than 'WWW' next to telnet://-links

### ISSUE THREAD

**Issue by Gapstare** [demozoo/demozoo#536]
What it says on the tin :slightly_smiling_face: 
Example: https://demozoo.org/bbs/273/

**Comment by sagamusix** [comment]
Essentially a dupe of #414

**Comment by gasman** [comment]
@sagamusix Not really - `telnet://` is something that we can pattern-match for, like we do for other 'recognised' link types. It doesn't require a manually-entered label, which is what #414 is asking for.

**Comment by sagamusix** [comment]
Right, I read too much into the "something other" and assumed that a custom label was wanted.

**Comment by gasman** [comment]
Completed in #660.
